jcl/src/java.base/share/classes/com/ibm/jit/JITHelpers.java

```diff
@@ -54,9 +54,6 @@ private JITHelpers() {
 
 	@CallerSensitive
 	public static JITHelpers getHelpers() {
-		if (Reflection.getCallerClass().getClassLoader() != null) {
-			throw new SecurityException("JITHelpers"); //$NON-NLS-1$
-		}
```

Comment:
@andrewcraik wanted to point this one out to you. Is this safe? We need a handle to the JITHelpers object in Test_JITHelpers.java so we can do unit testing on the newly added APIs and have full control over expected inputs and outputs to test all corner cases. Unfortunately we cannot get a handle to the object without removing this or going the reflection route which seems a bit dodgy. Are you aware of any reason why this protection was put in place?

Reply:
I believe this is here for the same reason sun.misc.Unsafe has the same kind of check - to dissuade others from getting the instance easily. Using a trick similar to unsafe to get hold of the instance in the tests would seem better to me than exposing the API in general.

Reply:
Cool, that makes sense. @dhong44 can you use a similar trick we use in String.java to get a hold of the JITHelpers object and revert the above deleted lines?